share/tools/web_config/webconfig.py

```diff
@@ -589,16 +600,18 @@ def get_tparm(key):
             g_special_escapes_dict["exit_attribute_mode"] = get_tparm("sgr0")
             g_special_escapes_dict["bold"] = get_tparm("bold")
             g_special_escapes_dict["underline"] = get_tparm("smul")
+
+            # If get_tparm return None then use default value
+            for key in g_default_special_escapes_dict:
+                if g_special_escapes_dict[key] is None:
+                    g_special_escapes_dict[key] = g_default_special_escapes_dict[key]
```

Comment:
This isn't useful.

If the terminfo entry doesn't include these things, then we can't extract them, but e.g. `set_color` also won't add them.

These hardcoded escapes are, if the terminfo entry is that awkward, not guaranteed to mean what this thinks they do.

This isn't specific to Windows, it can also happen e.g. if you set $TERM to "dumb".

Please remove this change.

Reply:
ok. I will remove this